src/main/java/com/minecolonies/coremod/colony/TriggerColonyAchievements.java

```diff
@@ -0,0 +1,232 @@
+                break;
+            case TAG_FISHERMAN_FISH:
+                colony.triggerAchievement(ModAchievements.achievementCatchOneFish);
+                break;
+            case TAG_FARMER_WHEAT:
+                colony.triggerAchievement(ModAchievements.achievementHarvestOneWheat);
+                break;
+            case TAG_FARMER_POTATOES:
+                colony.triggerAchievement(ModAchievements.achievementHarvestOnePotato);
+                break;
+            case TAG_FARMER_CARROTS:
+                colony.triggerAchievement(ModAchievements.achievementHarvestOneCarrot);
+                break;
+            case TAG_LUMBERJACK_SAPLINGS:
+                colony.triggerAchievement(ModAchievements.achievementPlantOneSapling);
+                break;
+            case TAG_LUMBERJACK_TREES:
+                colony.triggerAchievement(ModAchievements.achievementFellOneTree);
+                break;
+            default:
+                break;
+        }
+    }
+    /**
+     * Trigger second achievement.
+     * @param statistic the statistic.
+     */
+    public static void triggerSecondAchievement(@NotNull String statistic, @NotNull Colony colony)
+    {
+        switch (statistic)
+        {
+            case TAG_GUARD_MOBS:
+                colony.triggerAchievement(ModAchievements.achievementKill25Mobs);
+                break;
+            case TAG_MINER_ORES:
+                colony.triggerAchievement(ModAchievements.achievementMine25Ores);
+                break;
+            case TAG_MINER_DIAMONDS:
+                colony.triggerAchievement(ModAchievements.achievementMine25Diamonds);
+                break;
+            case TAG_BUILDER_HUTS:
+                colony.triggerAchievement(ModAchievements.achievementBuild25Huts);
+                break;
+            case TAG_FISHERMAN_FISH:
+                colony.triggerAchievement(ModAchievements.achievementCatch25Fish);
+                break;
+            case TAG_FARMER_WHEAT:
+                colony.triggerAchievement(ModAchievements.achievementHarvest25Wheat);
+                break;
+            case TAG_FARMER_POTATOES:
+                colony.triggerAchievement(ModAchievements.achievementHarvest25Potatoes);
+                break;
+            case TAG_FARMER_CARROTS:
+                colony.triggerAchievement(ModAchievements.achievementHarvest25Carrots);
+                break;
+            case TAG_LUMBERJACK_SAPLINGS:
+                colony.triggerAchievement(ModAchievements.achievementPlant25Saplings);
+                break;
+            case TAG_LUMBERJACK_TREES:
+                colony.triggerAchievement(ModAchievements.achievementFell25Trees);
+                break;
+            default:
+                break;
+        }
+    }
+    /**
+     * Trigger third achievement.
+     * @param statistic the statistic.
+     */
+    public static void triggerThirdAchievement(@NotNull String statistic, @NotNull Colony colony)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') The Cyclomatic Complexity of this method "triggerThirdAchievement" is 12 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Document the parameter(s): colony [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)